olm/check-yq.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
copyright date is wrong. use 2019 only

Reply:
fixed in https://github.com/eclipse/che-operator/pull/60/commits/cef9501b018e6eef240a80941c9fb8b6b7417624